include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort.h

```diff
@@ -146,7 +146,7 @@ template <typename _KernelName, ::std::uint32_t __radix_bits, bool __is_ascendin
 sycl::event
 __radix_sort_count_submit(_ExecutionPolicy&& __exec, ::std::size_t __segments, ::std::size_t __block_size,
                           ::std::uint32_t __radix_offset, _ValRange&& __val_rng, _CountBuf& __count_buf,
-                          sycl::event __dependency_event
+                          sycl::event __dependency_event, _Proj __proj
```

Comment:
Should `_dependency_event` go last?

Reply:
There is no a strict parameters order. These function are "internal" in the file and we can define order how we want (easy way).

Reply:
My idea was that everything that is related to the algorithm goes first and the dependencies go last. It's more or less about the logic of the parameters placing and correlation with the async API we currently have. Anyway, I don't want to insist.

Reply:
A separated issue was created 
https://github.com/oneapi-src/oneDPL/issues/856